saleor/json_schemas/types/TransactionChargeRequested.d.ts

```diff
@@ -0,0 +1,92 @@
+/* eslint-disable */
+/**
+ * This file was automatically generated by json-schema-to-typescript.
+ * DO NOT MODIFY IT BY HAND. Instead, modify the source JSONSchema file,
+ * and run json-schema-to-typescript to regenerate this file.
+ */
+
+export type TransactionChargeRequested =
+  | TransactionChargeRequestedSyncSuccess
+  | TransactionChargeRequestedSyncFailure
+  | TransactionChargeRequestedAsync;
+/**
+ * Psp reference received from payment provider.
+ */
+export type Pspreference = string;
+/**
+ * Decimal amount of the processed action
+ */
+export type Amount = number | string;
+/**
+ * Time of the action in ISO 8601 format
+ */
+export type Time = string;
+/**
+ * External url with action details
+ */
+export type Externalurl = string;
+/**
+ * Message related to the action. The maximum length is 512 characters; any text exceeding this limit will be truncated
+ */
+export type Message = string;
+export type Actions = ("CHARGE" | "REFUND" | "CANCEL")[] | null;
+/**
+ * Result of the action
+ */
+export type Result = "CHARGE_SUCCESS";
+/**
+ * Psp reference received from payment provider.
+ */
+export type Pspreference1 = string;
+/**
+ * Decimal amount of the processed action
+ */
+export type Amount1 = number | string;
+/**
+ * Time of the action in ISO 8601 format
+ */
+export type Time1 = string;
+/**
+ * External url with action details
+ */
+export type Externalurl1 = string;
+/**
+ * Message related to the action. The maximum length is 512 characters; any text exceeding this limit will be truncated
+ */
+export type Message1 = string;
+export type Actions1 = ("CHARGE" | "REFUND" | "CANCEL")[] | null;
+/**
+ * Result of the action
+ */
+export type Result1 = "CHARGE_FAILURE";
+/**
+ * Psp reference received from payment provider.
+ */
+export type Pspreference2 = string;
+export type Actions2 = ("CHARGE" | "REFUND" | "CANCEL")[] | null;
```

Comment:
I guess transpiler had a name conflict so it created "Actions2".

Would be good if we can have specific names (Actions2 is ActionsAsync for example)